.workshop/build

```diff
@@ -15,12 +15,6 @@ mv workshop /opt/app-root/workshop
 
 rm -f Dockerfile README.md LICENSE
 
-# If the workshop requires Git, it is necessary to set some defaults for
```

Comment:
We should not need these as the only git repository needed will be handled by the `task` for the pipeline that checkouts out the source code needed.

Reply:
I wouldn't remove these, as we don't know how the workshop will evolve.

Reply:
I can add this back, but I will comment out the git commands.